### DISCUSSION TITLE
Two solvers in the same input file

### DISCUSSION THREAD

**Discussion by steam-simulator** [idaholab/moose#26066] [Q&A Modules: General] [answered]
Hello,
Is it possible to use two different solvers in the same input file? For example, I want to use PJFNK for tensor mechanics and Newton for Porousflow in the same input file . Is there any example in this regard? Thanks.

**Comment by grmnptr** [comment, marked as answer]
As far as I know we don't have that capability right now. We just started supporting segregared nonlinear systems but the executioner support is not there yet. You can split the two systems using multiapps and then define the different solvers in each app.
